pkg/metadata/provider_vultr.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path"
+	"time"
+)
+
+const (
+	vultrMetaDataURL = "http://169.254.169.254/v1/"
+)
```

Comment:
Not exactly. There is a `userdata` field contained in it's metadata, but that's only valid when running a prebuilt VPS system. When using an ISO of any kind the server ignores it, making its value for LinuxKit rather dubious.